packages/template-block/src/components/CustomButton.tsx

```diff
@@ -34,7 +36,7 @@ export const CustomButton = ({
                 value={hasRichTextValue(buttonText) ? buttonText : convertToRteValue('p', 'Use this Template')}
```

Comment:
One line up the id of the rte should be unique so please add the blockId (not that important as here are no Plugins used, but maybe in the future...)